firebase_admin/auth.py

```diff
@@ -14,48 +14,50 @@
 
```

Comment:
Can we split up the contents of this file?  Maybe the validator and the user manager can go in a separate file?